src/Servers/Kestrel/Core/src/Internal/Infrastructure/KestrelTrace.cs

```diff
@@ -117,6 +117,10 @@ internal class KestrelTrace : IKestrelTrace
             LoggerMessage.Define<string>(LogLevel.Debug, new EventId(40, nameof(Http2MaxConcurrentStreamsReached)),
                 @"Connection id ""{ConnectionId}"" reached the maximum number of concurrent HTTP/2 streams allowed.");
 
+        private static readonly Action<ILogger, Exception> _invalidResponseHeaderRemoved =
+            LoggerMessage.Define(LogLevel.Information, new EventId(41, nameof(InvalidResponseHeaderRemoved)),
```

Comment:
nit: in the rest of the stack we try to avoid using `nameof` for the `EventId` because changing the name is technically a breaking change and it's easier to accidentally change it when renaming a method.

Reply:
Yeah, not sure why all of the Kestrel ones use nameof. Changing them is out of scope here.